src/main/java/duke/enums/Tasks.java

```diff
@@ -0,0 +1,34 @@
+package duke.enums;
```

Comment:
You already did an amazing job for Javadocs and I am aware that the requirement was to add documentation to half of the files, however I just felt that it would be even better to add some documentation to this file if time persists so that it is easier to understand the code.

Reply:
Noted, I will add more Javadocs if I have the time.